upload_to_s3.py

```diff
@@ -22,11 +22,14 @@
 SERVICE = os.getenv("SERVICE")
 BUCKET = os.getenv("BUCKET")
 
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
```

Comment:
i guess these env vars were just implied before, b/c `boto3` will grab them from the env on its own? in any case, i like this more fore sure!